common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -287,37 +288,20 @@ def __init__(
         """
         Create & open the connection, authenticate, and provide pointers to the collections
         """
-        if kwargs.get('replicaSet') is None:
-            kwargs.pop('replicaSet', None)
-            mongo_class = pymongo.MongoClient
-        else:
-            mongo_class = pymongo.MongoReplicaSetClient
-        _client = mongo_class(
-            host=host,
-            port=port,
-            tz_aware=tz_aware,
-            **kwargs
-        )
-        self.database = MongoProxy(
-            pymongo.database.Database(_client, db),
-            wait_time=retry_wait_time
-        )
+        # Set a write concern of 1, which makes writes complete successfully to the primary
+        # only before returning. Also makes pymongo report write errors.
+        kwargs['w'] = 1
```

Comment:
I don't see anything that is overriding this right now in our secure config.  We should make a separate story for making this configurable?  1 seems like the wrong write concern for durability, `majority' would be a better option as a default I feel but it will make mongo writes slightly slower.